README.md

```diff
@@ -97,13 +97,31 @@ You can find those example applications under `examples/`.
 
 ### Run the Trace Server
 
-In order to open traces, you need a trace server running on the same machine as the trace extension. You can download the [Eclipse Trace Compass server](https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/?d) or let `yarn` download and run it:
+In order to open traces, you need a trace server running on the same machine as the trace extension. There are 3 options to run the trace server.
+
+#### Using yarn (Linux only)
+
+If you are working on Linux, you can let `yarn` download and run it:
 
 ```bash
 yarn download:server
 yarn start:server
```

Comment:
First of all, if someone start the server externally, then that user has to stop by itself. Starting it manually, has the advantage, that the open-traces are populated with traces on the server, or when starting the server in debug mode from the server development environment to debug the server.

Secondly, the theia-trace-extension has a feature to automatically start the trace server from the application. There is a command or when trying to open a trace it will start it. That started server will be stopped by the applicaton as well.  By default, it uses the location where `yarn download:server` stores the server for both the browser backend and electron app.

If a user wants to use a different server build, the there is a preference in the application where the user can change the path to the custom server which will be used when starting it automatically. 

Could you please describe in this PR the trace viewer preference? Also describe the Start/stop trace server command (from the command menu opened using F1 or CTRL+SHIFT+P)

Reply:
> If a user wants to use a different server build, the there is a preference in the application where the user can change the path to the custom server which will be used when starting it automatically.

By this, do you mean what is described [here](https://github.com/theia-ide/theia-trace-extension#change-the-trace-server-url)?